armi/reactor/blueprints/gridBlueprint.py

```diff
@@ -179,6 +179,9 @@ class GridBlueprint(yamlize.Object):
     gridContents : dict
         A {(i,j): str} dictionary mapping spatialGrid indices in 2-D to string specifiers of what's
         supposed to be in the grid.
+    orientationBOL : dict
+        A {(i,j): float} dictionary mapping spatialGrid indices in 2-D to the orientation of
+        what's supposed to be in the grid.
```

Comment:
I want to confirm, this new parameter does not actually cause any rotation to take place, right? From looking at the code, my understanding is that the assembly is put in at whatever rotation that it is defined in the blueprints, and this new value is simply a bookkeeping parameter.

For instance, if I have an asymmetric assembly that is defined with a block grid pointed "up" (whatever that means), my understanding is that the reactor will be instantiated with that assembly pointed "up" no matter if `orientationBOL` is 0, 1, 2, 3, etc. The use of `orientationBOL` is simply to say that pointed "up" corresponds to an orientation value of `x`.

Is my understanding correct?

Reply:
That's a good point. I have adding in rotating the assemblies instead of just setting the `orientation` parameter.

I guess I was hyper-focused on a potential issue with 1/3 core conversions and got blind. Thanks for the assist.

Reply:
fixed